packages/loopring_v3/contracts/core/impl/libtransactions/TransferTransaction.sol

```diff
@@ -65,9 +67,12 @@ library TransferTransaction
         Transfer memory transfer = readTx(data, offset);
         TransferAuxiliaryData memory auxData = abi.decode(auxiliaryData, (TransferAuxiliaryData));
 
-        // Check validUntil
-        require(ctx.timestamp < auxData.validUntil, "TRANSFER_EXPIRED");
+        // Fill in withdrawal data missing from DA
```

Comment:
I some cases certain fields in TransferAuxiliaryData are not required. For example. The `maxFee` is not required if the transfer is signed off-chain using EdDSA signature. By using `abi.decode(auxiliaryData, (TransferAuxiliaryData));` we have to use 96bits anyway, correct?